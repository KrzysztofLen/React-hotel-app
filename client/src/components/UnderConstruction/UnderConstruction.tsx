import React from 'react';
import {Link} from "react-router-dom";

const UnderConstruction = () => (
    <div className={"underConstruction"}>
        <h1 className={"underConstruction__header"}>Under Construction</h1>
        <h2 className={"underConstruction__subHeader"}>Content will be available soon</h2>
        <Link to={"/"} className={"button is-primary"}>Home</Link>
    </div>
);

export default UnderConstruction;

