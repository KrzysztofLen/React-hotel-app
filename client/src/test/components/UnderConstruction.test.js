import React from 'react';
import {shallow} from 'enzyme';
import UnderConstruction from "../../components/UnderConstruction/UnderConstruction";

test('should render UnderConstruction component correctly', () => {
	const wrapper = shallow(<UnderConstruction />);
	expect(wrapper.debug()).toMatchSnapshot();
});