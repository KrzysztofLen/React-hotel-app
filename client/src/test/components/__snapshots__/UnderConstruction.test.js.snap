// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render UnderConstruction component correctly 1`] = `
"<div className=\\"underConstruction\\">
  <h1 className=\\"underConstruction__header\\">
    Under Construction
  </h1>
  <h2 className=\\"underConstruction__subHeader\\">
    Content will be available soon
  </h2>
  <Link to=\\"/\\" className=\\"button is-primary\\">
    Home
  </Link>
</div>"
`;
